static-site/src/components/splash/errorMessages.jsx

```diff
@@ -27,3 +27,10 @@ export const ErrorBanner = ({title, contents}) => (
     <p>{contents}</p>
   </FixedBanner>
 );
+
+export const DataFetchErrorParagraph = () => (
+  <splashStyles.CenteredFocusParagraph>
+    Something went wrong getting data.
+    Please <a href="mailto:hello@nextstrain.org">contact us at hello@nextstrain.org </a> if this continues to happen.
+  </splashStyles.CenteredFocusParagraph>
+);
```

Comment:
I did this refactoring thinking I would be using the same error paragraph in the new page, but I ended up not using it. Keeping this commit in here anyways since it's nice to consolidate the errors.